go.work

```diff
@@ -1,5 +1,6 @@
 go 1.22
 
+
```

Comment:
Is this whitespace change neede?

Reply:
This was inadvertent and has been removed.